pyvista/core/filters/poly_data.py

```diff
@@ -368,35 +384,42 @@ def intersection(poly_data, mesh, split_first=True, split_second=True):
             The mesh to intersect with.
 
         split_first : bool, optional
-            If `True`, return the first input mesh split by the intersection with the
-            second input mesh.
+            If ``True``, return the first input mesh split by the
+            intersection with the second input mesh.
 
         split_second : bool, optional
-            If `True`, return the second input mesh split by the intersection with the
-            first input mesh.
+            If ``True``, return the second input mesh split by the
+            intersection with the first input mesh.
 
         Returns
         -------
         intersection: pyvista.PolyData
             The intersection line.
 
         first_split: pyvista.PolyData
-            The first mesh split along the intersection. Returns the original first mesh
-            if `split_first` is False.
+            The first mesh split along the intersection. Returns the
+            original first mesh if `split_first` is ``False``.
 
         second_split: pyvista.PolyData
-            The second mesh split along the intersection. Returns the original second mesh
-            if `split_second` is False.
+            The second mesh split along the intersection. Returns the
+            original second mesh if `split_second` is ``False``.
```

Comment:
This SO question/answer does a good job of explaining it:
https://stackoverflow.com/questions/22256995/restructuredtext-in-sphinx-and-docstrings-single-vs-double-back-quotes-or-back

Effectively, by default a single backtick is undefined by default and will be rendered as an inline literal.  We could go with either one since we don't define the definition of it and it's automatically an inline literal.  Double backticks ensure that it's an inline literal, so I'd go with those.

Reply:
But they do get rendered differently. Single backticks are italic, double backticks are pink fixed font with a background shade.

Reply:
Ok.  Then we go with double backquotes.  Didn't realize they were getting rendered differently.  Thanks for the attention to detail.